content/src/roadmaps/raffle-bingo-steps/raffle-license-step-1.md

```diff
@@ -1,17 +1,12 @@
 ---
 notesMd: "[2526: Add non-essential question tasks: raffle task screen](https://dev.azure.com/NJInnovation/Business%20First%20Stop/_workitems/edit/2526)"
 id: "raffle-license-step-1"
-webflowId: "66be3c5382fc36dffc2ce314"
```

Comment:
When you merge this, remind me so that we can remove these from Webflow. I think the sync job will take care of this, but just incase we can proactively delete them.